src/pages/AboutPage/AboutPage.jsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
-
+import AboutUs from '../../components/AboutUs/AboutUs';
 import PortalLayout from '../../layouts/PortalLayout';
 
 const App = ({ location: { pathname } = {} } = {}) => (
-  <PortalLayout pathname={pathname}>TODO: About Page</PortalLayout>
+  <PortalLayout pathname={pathname}>
+    <AboutUs />
```

Comment:
Since the code for AboutPage is quite small, we don't have to create a new **Components** for AboutUs. You can transfer the code over here from the AboutUs components back to AboutPage so that we minimise the number of components we make. Usually we make something a component when we think that it can be reused by other pages or that it makes up a small part of a Page.

Reply:
ok!